notebooks/concepts/Imviz_RadialPlot_Example.ipynb

```diff
@@ -0,0 +1,685 @@
+    "scat2d = glue_app.scatter2d(data='plotdat', x='rdat', y='dat')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "96db6ced",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "scat2d.figure_widget"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "394bb604",
+   "metadata": {},
+   "source": [
+    "# From simple aperture photometry plugin"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9439d322",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from jdaviz.configs.imviz.plugins.aper_phot_simple.aper_phot_simple import SimpleAperturePhotometry"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7a345924",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plg = SimpleAperturePhotometry(app=imviz.app)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8902b9ef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plg.vue_data_selected('acs_47tuc_1[SCI,1]')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8ee18830",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plg.vue_subset_selected('Subset 1')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "536226a0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This code is already inside the plugin\n",
+    "self = plg\n",
+    "data = self._selected_data\n",
+    "reg = self._selected_subset\n",
+    "comp = data.get_component(data.main_components[0])\n",
+    "bg = 0\n",
+    "comp_no_bg = comp.data - bg\n",
+    "aper_mask = reg.to_mask(mode='subpixels', subpixels=32)  # rectangle\n",
+    "img = aper_mask.get_values(comp_no_bg, mask=None)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2f1ebf3d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Sanity check: Should match Imviz viewer selection above.\n",
+    "plt.imshow(img.reshape(aper_mask.shape), vmin=46, vmax=287, origin='lower')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2760cfc2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This is new code that we need to add\n",
+    "reg_bb = reg.bounding_box\n",
+    "xx = np.ogrid[reg_bb.iymin:reg_bb.iymax, reg_bb.ixmin:reg_bb.ixmax]\n",
+    "dx = xx[1] - reg.center.x\n",
+    "dy = xx[0] - reg.center.y\n",
+    "r = np.hypot(dx, dy)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cebe4a5a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We can use the glue_app stuff here, but I just want to\n",
+    "# see if we get the same result in Matplotlib.\n",
+    "# Should look the same as \"Subset based selection\" below.\n",
+    "#\n",
+    "# Kyle, this is what will get sent to the frontend:\n",
```

Comment:
@kecnry , I think this is what we need to send to bqplot: `r.ravel(), img` (two 1D float arrays).

Maybe if you read the whole "From simple aperture photometry plugin" here, it would be clearer what I mean?

With the `glue_app` thingy that Erik originally implemented here, the radial plot is interactive. But adding data to `glue_app` is not ideal because these are throw-away buffers. We don't want to keep adding/deleting them. If there is a way to make your bqplot implementation interactive without glue_app, then great. If not, maybe it is not a big deal?

Reply:
Ok, I'll see if I can get bqplot in without your buffer concerns, and if not, we can proceed with mpl for now and re-consider down the road if we need interactivity or want more advanced functionality (adding to the main viewer, etc).

Reply:
In Ginga, the plot is static but you can adjust some settings and replot. For instance, the user would want to adjust the max of X-axis (zoom in/out horizontally) but there are several ways to accomplish that depending on your findings.

Reply:
(Actually recently Eric Jeschke might have added interactivity via key bindings but that is another story.)